<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>484</width>
    <height>498</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>200</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>LogViewer</string>
  </property>
  <widget class="QWidget" name="wdgCentral">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="layHeader" stretch="1,0">
      <item>
       <layout class="QHBoxLayout" name="layFile">
        <item>
         <widget class="QLabel" name="lblFile">
          <property name="text">
           <string>File :</string>
          </property>
          <property name="buddy">
           <cstring>cmbFile</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cmbFile">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="layPauseRefresh">
        <item>
         <widget class="QPushButton" name="butPause">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Pause</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="butRefresh">
          <property name="text">
           <string>Refresh</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTableView" name="tblLog"/>
    </item>
    <item>
     <widget class="QGroupBox" name="grpFilter">
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="title">
       <string>Filter</string>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="layFilterGlobalSelect">
         <item>
          <widget class="QPushButton" name="butFilterAll">
           <property name="text">
            <string>All</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="butNone">
           <property name="text">
            <string>None</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="spc">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="laySeverity">
         <item>
          <widget class="QLabel" name="lblSeverity">
           <property name="text">
            <string>Severity</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="layModule">
         <item>
          <widget class="QLabel" name="lblModule">
           <property name="text">
            <string>Module</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="layThread">
         <item>
          <widget class="QLabel" name="lblThread">
           <property name="text">
            <string>Thread</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="laySource">
         <item>
          <widget class="QLabel" name="lblSource">
           <property name="text">
            <string>Source</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
      <zorder></zorder>
      <zorder>lblThread</zorder>
      <zorder>lblThread</zorder>
      <zorder></zorder>
      <zorder></zorder>
      <zorder></zorder>
      <zorder></zorder>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="mnuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>484</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="mnuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actOpen"/>
   </widget>
   <addaction name="mnuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actOpen">
   <property name="text">
    <string>&amp;Open directory</string>
   </property>
   <property name="toolTip">
    <string>Open all log in a directory</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>cmbFile</tabstop>
  <tabstop>butPause</tabstop>
  <tabstop>butRefresh</tabstop>
  <tabstop>tblLog</tabstop>
  <tabstop>butFilterAll</tabstop>
  <tabstop>butNone</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
